lzy/lzy-service/src/main/java/ai/lzy/service/operations/start/StartExecution.java

```diff
@@ -37,12 +31,8 @@ private StartExecution(StartExecutionBuilder builder) {
         super(builder);
         this.storageConfig = builder.storageConfig;
         this.state = builder.state;
+        this.steps = List.of(createKafkaTopic(), StartExecution.this::complete);
```

Comment:
`createAllocatorSession()` step is missing

Reply:
I didn't add this step here to not create fictive sessions for now.